guide/13-managing-arcgis-applications/hub-premium-guide.ipynb

```diff
@@ -0,0 +1,471 @@
+{
```

Comment:
<p>First, we will sign in to an organization with Hub Premium enabled to demonstrate these workflows.  </p><p>Note: In order to replicate these workflows for yourself, you may have to verify the features available to you, per your ArcGIS Hub licensing.  </p><p>You may also want to look at these examples for site and page creation and cloning using <strong>ArcGIS Basic</strong> (insert live URL) and site and page layout and theme editing using <strong>Enterprise Sites</strong> (insert live URL) to see other methods of working with your Hub using the ArcGIS API for Python. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Please insert the appropriate relative links for the guides.